__utils__/tsconfig/tsconfig.json

```diff
@@ -12,7 +12,8 @@
     "removeComments": false,
     "sourceMap": true,
     "strict": true,
-    "target": "es2021"
+    "target": "es2021",
+    "skipLibCheck": true
```

Comment:
No idea why it kept complaining jest error 🤷 . I can't push to remote without it, feel free to remove before merging.